src/cr/chain_replication_service.cc

```diff
@@ -1,20 +1,21 @@
 #include "cr/chain_replication_service.h"
 #include "hashing_utils.h"
+#include "status_codes.h"
 #include "cr/chain_replication_utils.h"
 namespace smf {
 namespace chains {
 future<smf::rpc_envelope> chain_replication_service::mput(
   smf::rpc_recv_typed_context<tx_multiput> &&puts) {
   smf::rpc_envelope e(nullptr);
-  e.set_status(501); // Not implemented
+  e.set_status(HTTP_STATUS_NOT_IMPLEMENTED); // 501
```

Comment:
I'm fine keeping the 
```
HTTP_*
```

However, would one be bounded by the HTTP extensions. ... i guess not. 

Is it worth to do ```SMF_*``` ? 

I'm fine either way.